Makefile

```diff
@@ -32,6 +32,9 @@ bazel-push-images:
 
 push: bazel-push-images
 
+patch: 
+	hack/patch.sh
```

Comment:
Can you change this to `hack/dockerized "export BUILD_ARCH=${BUILD_ARCH} && DOCKER_PREFIX=${DOCKER_PREFIX} DOCKER_TAG=${DOCKER_TAG} DOCKER_TAG_ALT=${DOCKER_TAG_ALT} IMAGE_PREFIX=${IMAGE_PREFIX} IMAGE_PREFIX_ALT=${IMAGE_PREFIX_ALT} hack/patch.sh"`?

That should help to work more semaless together with all other targets.

Reply:
Done